test/fuzz/CMakeLists.txt

```diff
@@ -68,6 +68,7 @@ if (${SPIRV_BUILD_FUZZER})
           transformation_permute_function_parameters_test.cpp
           transformation_permute_phi_operands_test.cpp
           transformation_push_id_through_variable_test.cpp
+          transformation_replace_add_sub_mul_with_carrying_extended_test.cpp
```

Comment:
I would be grateful if you could sort these filenames alphabetically. Someone (probably me :) did something wrong.